lib/depject/sheet/display.js

```diff
@@ -5,9 +5,13 @@ exports.gives = nest('sheet.display')
 
 exports.create = function () {
   return nest('sheet.display', function (handler) {
-    const { content, footer, classList, onMount } = handler(done)
+    const { content, footer, classList, onMount, attributes } = handler(done)
 
-    const container = h('div', { className: 'Sheet', classList }, [
+    let fullAttributes = { className: 'Sheet', classList }
+    if (attributes !== undefined) {
+      fullAttributes = { ...attributes, ...fullAttributes }
+    }
+    const container = h('div', fullAttributes, [
```

Comment:
This feels like it could be cleaner. I'm passing the `classList` and the `attributes` separately just to have them be recombined...

Reply:
Also, could I maybe attache the event handlers (passed as `attributes`) to something else that makes more sense?